src/broadcast/workerInterfaces.ts

```diff
@@ -0,0 +1,114 @@
+import { dolphinManager } from "@dolphin/manager";
+import { app } from "electron";
+import electronLog from "electron-log";
+import { spawn, Thread, Worker } from "threads";
+
+import { Methods as BroadcastWorkerMethods, WorkerSpec as BroadcastWorkerSpec } from "./broadcastWorker";
+import {
+  ipc_broadcastErrorOccurredEvent,
+  ipc_broadcastListUpdatedEvent,
+  ipc_dolphinStatusChangedEvent,
+  ipc_slippiStatusChangedEvent,
+} from "./ipc";
+import { Methods as SpectateWorkerMethods, WorkerSpec as SpectateWorkerSpec } from "./spectateWorker";
+import { BroadcasterItem } from "./types";
+
+const log = electronLog.scope("broadcast/workerInterface");
+const broadcastLog = electronLog.scope("broadcastManager");
+const spectateLog = electronLog.scope("spectateManager");
+
+export const broadcastWorker: Promise<Thread & BroadcastWorkerMethods> = new Promise((resolve, reject) => {
+  log.debug("broadcast: Spawning worker");
+
+  spawn<BroadcastWorkerSpec>(new Worker("./broadcastWorker"), { timeout: 30000 })
+    .then((worker) => {
+      worker.getDolphinStatusObservable().subscribe(({ status }) => {
+        log.info(`got dolphin status: ${status}`);
```

Comment:
Should we remove these logs? I think they were useful for debugging but they're probably not useful now.

Reply:
yeah we can clean those up